app/controllers/split_checkout_controller.rb

```diff
@@ -63,6 +63,10 @@ def order_error_messages
 
     # Replace "Bill address phone" by "Customer phone" to improve readability
     replace_bill_address_phone_error if @order.errors["bill_address.phone"].present?
+
+    # Reorder errors to make sure the most important ones are shown first
+    # and finally, return the error messages to sentence
+    reorder_errors.map(&:full_message).to_sentence
```

Comment:
Nice work. I can imagine that the `form_for` helper could potentially do this for us, based on the order of elements in the form.
But it clearly doesn't, even the documentation shows the fields out of order :( (https://guides.rubyonrails.org/v2.3.8/activerecord_validations_callbacks.html#error-messages-and-error-messages-for )
I can't seem to find any mention of this online. I think maybe we could do this by overriding the [FormBuilder](https://apidock.com/rails/ActionView/Helpers/FormBuilder). It would be cool to have a generic solution. But meh, this works 👍